handsontable/src/editorManager.js

```diff
@@ -141,9 +141,9 @@ class EditorManager {
       return;
     }
 
-    const { highlight } = this.hot.getSelectedRangeLast();
+    const highlight = this.hot.getSelectedRangeLast()?.highlight;
 
-    if (highlight.isHeader()) {
+    if (!highlight || highlight.isHeader()) {
```

Comment:
```suggestion
    if (highlight === undefined || highlight.isHeader()) {
```

For me, it would be more clear.

Reply:
I can't entirely agree. With the origin expression, I catch bestially all false-ish values. It's safer.